src/main/kotlin/no/nav/syfo/application/api/PodApi.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.syfo.application.api
+
+import io.ktor.application.*
+import io.ktor.http.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import no.nav.syfo.application.ApplicationState
+import no.nav.syfo.application.database.DatabaseInterface
+
+fun Routing.registerPodApi(
+    applicationState: ApplicationState,
+    database: DatabaseInterface,
+) {
+    get("/internal/is_alive") {
+        if (applicationState.alive) {
+            call.respondText("I'm alive! :)")
+        } else {
+            call.respondText("I'm dead x_x", status = HttpStatusCode.InternalServerError)
+        }
+    }
+    get("/internal/is_ready") {
```

Comment:
Ta gjerne konstantaer for paths her og så kan de gjenbrukes i test. Har gjort det i den nyeste appen isoppfolgingstilfelle

```
const val podLivenessPath = "/internal/is_alive"
const val podReadinessPath = "/internal/is_ready"
```